mona_sdk/client.py

```diff
@@ -114,17 +116,21 @@
     "SAMPLING_FACTORS_MAX_AGE_SECONDS", 300
 )
 
-SERVICE_ERROR_MESSAGE = "Could not get server response for the wanted service."
-UPLOAD_CONFIG_ERROR_MESSAGE = (
-    "Could not upload the new configuration, please check it is valid."
+UNAUTHENTICATED_ERROR_CHECK_MESSAGE = (
+    f"Notice that should_use_authentication is set to False, which is not supported by"
```

Comment:
nit - these aren't f"" strings, its regular strings, right? I think it will make some linters be angry if you use f"" without placeholders, but I might be wrong